pkgs/development/libraries/nuraft/default.nix

```
@@ -0,0 +1,20 @@
+{ stdenv, fetchurl, cmake, boost172, asio, openssl, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "nuraft";
+  version = "1.1.2";
+
+  src = fetchurl {
+    url = "https://github.com/eBay/NuRaft/archive/v${version}.tar.gz";
+    sha256 = "sha256-VZT2q32Md48IkOmtj52P3fsF3jkW60WjFqoglITjHVM=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ boost172 asio openssl zlib ];
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/eBay/NuRaft";
+    description = "C++ implementation of Raft core logic as a replication library ";
```

Trailing whitespace.

```suggestion
    description = "C++ implementation of Raft core logic as a replication library";
```